src/shim/machine/manifest.rs

```diff
@@ -3,32 +3,42 @@
 // Copyright 2021-2023 Protocol Labs
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-use ahash::{HashMap, HashMapExt};
+use ahash::HashMap;
 use anyhow::{anyhow, Context};
 use cid::Cid;
 use fvm_ipld_blockstore::Blockstore;
 use fvm_ipld_encoding3::CborStore;
 
 // For details on actor name and version, see <https://github.com/filecoin-project/go-state-types/blob/1e6cf0d47cdda75383ef036fc2725d1cf51dbde8/manifest/manifest.go#L36>
 
-pub const ACCOUNT_ACTOR_NAME: &str = "account";
-pub const CRON_ACTOR_NAME: &str = "cron";
-pub const INIT_ACTOR_NAME: &str = "init";
-pub const MARKET_ACTOR_NAME: &str = "storagemarket";
-pub const MINER_ACTOR_NAME: &str = "storageminer";
-pub const MULTISIG_ACTOR_NAME: &str = "multisig";
-pub const PAYCH_ACTOR_NAME: &str = "paymentchannel";
-pub const POWER_ACTOR_NAME: &str = "storagepower";
-pub const REWARD_ACTOR_NAME: &str = "reward";
-pub const SYSTEM_ACTOR_NAME: &str = "system";
-pub const VERIFREG_ACTOR_NAME: &str = "verifiedregistry";
-// actor version >= 9
-pub const DATACAP_ACTOR_NAME: &str = "datacap";
-// actor version >= 10
-pub const EVM_ACTOR_NAME: &str = "evm";
-pub const EAM_ACTOR_NAME: &str = "eam";
-pub const PLACEHOLDER_ACTOR_NAME: &str = "placeholder";
-pub const ETH_ACCOUNT_ACTOR_NAME: &str = "ethaccount";
+macro_rules! name {
```

Comment:
This should most likely be replaced by, e.g., https://github.com/ChainSafe/fil-actor-states/blob/main/fil_actors_shared/src/v11/runtime/builtins.rs

It can be a separate PR, though.

Reply:
#3076